net/unbound/files/README.md

```diff
@@ -8,82 +8,170 @@ Unbound may be useful on consumer grade embedded hardware. It is *intended* to b
+		option port '53'
+		list server '127.0.0.1#1053'
+		list server '::1#1053'
+		...
+
+### Parallel dnsmasq
+In this case, Unbound serves your local network directly for all purposes. It will look over to dnsmasq for DHCP-DNS resolution. Unbound is generally accessible on port 53, and dnsmasq is only accessed at 127.0.0.1:1053 by Unbound. Although you can dig/drill/nslookup remotely with the proper directives.
 
 **/etc/config/unbound**:
 
 	config unbound
-		option dnsmasq_link_dns '1'
+		option dhcp_link 'dnsmasq'
+		option listen_port '53'
 		...
 
 **/etc/config/dhcp**:
 
 	config dnsmasq
-		option option noresolv '1'
-		option resolvfile '<empty>'
+		option domain 'yourdomain'
+		option noresolv '1'
+		option resolvfile ''
 		option port '1053'
 		...
 
-	config dhcp '<name>'
+	config dhcp 'lan'
 		list dhcp_option 'option:dns-server,0.0.0.0'
 		...
 
-Alternatives are mentioned here for completeness. DHCP event scripts which write host records are difficult to formulate for Unbound, NSD, or Bind. These programs sometimes need to be forcefully reloaded with host configuration, and reloads can bust cache. **Serial** configuration between dnsmasq and Unbound can be made on 127.0.0.1 with an off-port like #1053. This may double cache storage and incur unnecessary transfer delay.
+### Only odhcpd
+Why use dnsmasq you might ask? Well test, try, and review. You can have Unbound and odhcpd only. **You must install unbound-control.** When odhcpd configures as DHCP lease, it will call a script. The script provided with Unbound will read the lease file and enter DHCP-DNS records as much as dnsmasq once did.
+
+**/etc/config/unbound**:
+
+	config unbound
+		# name your router in DNS
+		option add_local_fqdn '1'
+		option add_wan_fqdn '1'
+		option dhcp_link 'odhcpd'
+		# add SLAAC inferred from DHCPv4
+		option dhcp4_slaac6 '1'
+		option domain 'lan'
+		option domain_type 'static'
+		option listen_port '53'
+		option rebind_protection '1'
+		# install unbound-control and set this
+		option unbound_control '1'
+		...
+
+**/etc/config/dhcp**:
+
+	config dhcp 'lan'
+		option dhcpv4 'server'
+		option dhcpv6 'server'
+		option interface 'lan'
+		# short times help renew events to refresh dns
+		option leasetime '4h'
+		option ra 'server'
+		option ra_management '1'
+		# issue your ULA and avoid default [fe80::]
+		list dns 'fdxx:xxxx:xxxx::1'
+
+	config odhcpd 'odhcpd'
+		option maindhcp '1'
+		option leasefile '/var/lib/odhcpd/dhcp.leases'
+		# this is where the magic happens
+		option leasetrigger '/usr/lib/unbound/odhcpd.sh'
 
 ## Back to Manual Configuration
-You don't want UCI, but don't worry. We have UCI for that. However, OpenWrt or LEDE are targeted at embedded machines with flash ROM. The initialization scripts do a few things to protect flash ROM. 
+You do not want UCI. We have UCI for that. However, OpenWrt or LEDE are targeted at embedded machines with flash ROM. The initialization scripts do a few things to protect flash ROM.
```

Comment:
Yes. I am actively chopping the documentation down. It may get uglier before it gets better.